packages/core/usePointerSwipe/index.ts

```diff
@@ -101,8 +101,8 @@ export function usePointerSwipe(
       // Disable scroll on for TouchEvents
       targetRef.value?.style?.setProperty('touch-action', 'none')
       // Future pointer events will be retargeted to target until pointerup/cancel
-      targetRef.value?.setPointerCapture(e.pointerId)
-
+      const eventTarget = e.target as HTMLElement | undefined
+      eventTarget?.setPointerCapture(e.pointerId)
```

Comment:
```suggestion
        (e.target as HTMLElement | undefined)?.setPointerCapture(e.pointerId)
```

Reply:
I don't actually mind to have two lines for type assertion. But I am not so sure of the logic here, does it make sense to capture on the event target instead @lstoeferle?

Reply:
In the end `e.target` is the same element like `targetRef.value`. Maybe there is some browser specific implementation that breaks the click events on chrome (black-box for me).